tools/osbuild-mpp

```diff
@@ -407,7 +407,8 @@ class ImageManifest:
         self.digest = "sha256:" + hashlib.sha256(raw).hexdigest()
 
     def is_manifest_list(self):
-        return self.media_type in ("application/vnd.docker.distribution.manifest.list.v2+json", "application/vnd.oci.image.index.v1+json")
+        return self.media_type in ("application/vnd.docker.distribution.manifest.list.v2+json", "application/vnd.oci.image.index.v1+json") or \
+            self.media_type == "" and self.json.get("manifests") is not None
```

Comment:
I think I prefer splitting this, rather than using the python-line-breaks. Something like this:

```python
        if self.media_type in ("application/vnd.docker.distribution.manifest.list.v2+json", "application/vnd.oci.image.index.v1+json")
            return True
        if not self.media_type and self.json.get("manifests") is not None:
            return True
        return False
```

And then you can also add comments above each conditional if it requires further elaboration.